umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-entire-umh.md

```diff
@@ -0,0 +1,123 @@
++++
+title = "Backup and Restore the United Manufacturing Hub"
+menuTitle = "Backup and Restore the UMH"
+description = "This page describes how to backup and restore the entire UMH."
+weight = 1000
++++
+
+This page describes how to back up the following:
+ - All Node-RED flows
+ - All Grafana dashboards
+ - The current Helm values for the united-manufacturing-hub chart
+ - All timescale tables used by the united-manufacturing-hub chart
+
+It does **not** back up:
+ - Other grafana data like Alerts, Users, ...
+ - Timescale data unrelated to the united-manufacturing-hub chart 
+ - Any other data in the cluster 
+ - TimescaleDB continuous aggregates: If you want to back them up, please follow [this guide](https://docs.timescale.com/self-hosted/latest/migration/schema-then-data/#recreate-continuous-aggregates)
+ - TimescaleDB policies: If you want to back them up, please follow [this guide](https://docs.timescale.com/self-hosted/latest/migration/schema-then-data/#recreate-policies)
+    
+
+## Requirements
+
+1. Stop stateful services -> kafka/redpanda and HiveMQ to prevent inconsistencies in the dataset.
+
+   ![Untitled](/images/production-guide/backup_recovery/backup-entire-umh/lensStopStatefulServices.png?width=75%)
+2. Create a Grafana API Token for an admin user:
+   - login to Grafana
+   - click on **settings** -> **API keys**
+   
+     ![Untitled](/images/production-guide/backup_recovery/backup-entire-umh/grafanaSettingsApiKey.png?)
+   - create a new API-key and change its role to **Admin**.
+   
+     ![Untitled](/images/production-guide/backup_recovery/backup-entire-umh/ApiKeycahngeRole.png?)
+   
+3. Download the [backup](https://github.com/united-manufacturing-hub/backup) repository. 
+4. Make sure you have postgresql installed.
+
+
+
+## Backup
```

Comment:
Divide this section into subsection using h3 headings like `### Generate Grafana API Key`, `### Stop workloads` and `### Backup using the script` (in this order). Try to avoid using images, and use words instead. Use the `resource` shortcode to give the full name of the StatefulSets that the user need to scale down.

```markdown
### Generate Grafana API Key

First, you need to generate a Grafana API Key with the admin role. Follow these steps...
1. ...

### Stop workloads

To prevent data inconsistencies, you need to temporarily stop the MQTT and Kafka Brokers. 
1. In Lens* go to the page...
2. Select the kafka* statefulset...
3. _use here the `include` shortcode_
4. do stuff...
5. Repeat the process for the mqtt* 

### Backup using the script
...
```
- [*]: remember to use the `resource` shortcode
- [`include`](https://umh.docs.umh.app/docs/development/contribute/documentation/style/hugo-shortcodes/#include) shortcode